bin/i18n/utils/malformed_i18n_reporter.rb

```diff
@@ -51,7 +51,7 @@ def report
       def google_drive
         return @google_drive if defined? @google_drive
 
-        @google_drive = CDO.gdrive_export_secret && Google::Drive.new(service_account_key: CDO.gdrive_export_secret)
+        @google_drive = CDO.gdrive_export_secret && Google::Drive.new
```

Comment:
tiny nit: I'd love it if this method didn't even need to know the exact name of the key any more; what do you think about adding a method on the `Google::Drive` class? Something like,

```ruby
def self.service_account_key
  StringIO.new(CDO.gdrive_export_secret&.to_json)
end
```

We could use it in the initialize method to make it even simpler than it already is, and use it here like

```suggestion
        @google_drive = Google::Drive.service_account_key.present? && Google::Drive.new
```

Reply:
I’m wary of tinkering with this i18n stuff, but it seems to me like the check for the Secret is unnecessary. This feature is going to raise an error if the system running this i18n code doesn’t have access to the Secret. It’s just a question of whether that happens during instantiation of `Google::Drive` or a later attempt to call methods on `@google_drive`.

Reply:
Lol, yeah, good point